src/libsystemd/sd-bus/busctl.c

```diff
@@ -67,7 +67,7 @@ static usec_t arg_timeout = 0;
 
 static int list_bus_names(sd_bus *bus, char **argv) {
         _cleanup_strv_free_ char **acquired = NULL, **activatable = NULL;
-        _cleanup_free_ char **merged = NULL;
+        _cleanup_free_ /* sic */ char **merged = NULL;
```

Comment:
I guess the question is how explicit we need to be. If you think "sic" is not clear, I'll change it.